test/TesterInternal/StorageTests/SQLAdapter/temp/TestEnvironmentInformation.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace UnitTests.StorageTests.SQLAdapter.temp
+{
+    /// <summary>
+    /// Enumeration of the supported test environment configurations.
+    /// </summary>
+    public enum SupportedTestConfigurations
+    {
+        None                        = 0,
+        Development                 = 1,
+        Jenkins                     = 2,
+        VisualStudioTeamServices    = 3
+    }
```

Comment:
Hmmm, this is something we should not have... I might want to run a different combination of tests regardless of where it is actually running...